src/api/Customer/getCustomerOffers.ts

```diff
@@ -0,0 +1,49 @@
+import { getData } from 'util/appConfigHelper';
+import fetchWithJWT from 'util/fetchHelper';
+import getApiURL from 'util/environmentHelper';
+import jwtDecode from 'jwt-decode';
+import { PaymentGateway } from 'redux/publisherConfigSlice';
+
+type CustomersOffer = {
+  expiresAt: number;
+  externalPaymentId: string;
+  inTrial: boolean;
+  nextPaymentAt: number;
+  nextPaymentCurrency: string;
+  nextPaymentPrice: number;
+  offerId: string;
+  offerTitle: string;
+  offerType: string;
+  paymentGateway: PaymentGateway;
+  paymentMethod: string;
+  pendingSwitchId: unknown;
+  period: string;
+  startedAt: number;
+  status: string;
+  subscriptionId: number;
+  totalPrice: number;
+};
```

Comment:
getCustomerOffers returns also passes which doesn't have all these fields. Worth to test and add also passes types